app/Main.hs

```diff
@@ -60,7 +68,11 @@ mvActualToGolden testPath =
    in do
      actualFileExist <- doesFileExist actualFilePath
      when actualFileExist (do
-       putStrLn $ "  Replacing file: " ++ goldenFilePath ++ " with: " ++ actualFilePath
+       putStr "  Replacing file: "
+       putStrColor Yellow goldenFilePath
+       putStr " with: "
+       putStrColor Green actualFilePath
+       putStrLn ""
        renameFile actualFilePath goldenFilePath)
```

Comment:
```suggestion
       renameFile actualFilePath goldenFilePath `catch` handleErr)
 where
  handleErr e = putStrColor Red $ "Warning: Could not replace file due to error: " ++ show (e :: IOException)
  ```
  
I noticed that this may [fail](https://hackage.haskell.org/package/directory-1.3.8.2/docs/System-Directory.html#v:renameFile). For example, `renameFile` may fail because the file doesn't exist or because of a permissions issue. What do you think about adding some error handling? 😄

Reply:
Hey @FranzGB That's a great suggestion! Thank you.